ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -475,26 +482,26 @@ typedef enum
--(void)documentController:(PTDocumentController*)documentController annotationsSelected:(NSString*)annotations;
--(void)documentController:(PTDocumentController*)documentController formFieldValueChanged:(NSString*)fieldString;
--(void)documentController:(PTDocumentController*)docVC behaviorActivated:(NSString*)behaviorString;
--(void)documentController:(PTDocumentController *)docVC leadingNavButtonClicked:(nullable NSString *)nav;
--(void)documentController:(PTDocumentController*)docVC longPressMenuPressed:(NSString*)longPressMenuPressedString;
--(void)documentController:(PTDocumentController *)docVC annotationMenuPressed:(NSString*)annotationMenuPressedString;
--(void)documentController:(PTDocumentController *)docVC leadingNavButtonClicked:(nullable NSString *)nav;
--(void)documentController:(PTDocumentController *)docVC pageChanged:(NSString*)pageNumbersString;
--(void)documentController:(PTDocumentController *)docVC zoomChanged:(NSNumber*)zoom;
--(void)documentController:(PTDocumentController *)docVC pageMoved:(NSString*)pageNumbersString;
--(void)documentController:(PTDocumentController *)docVC scrollChanged:(NSString*)scrollString;
+- (void)documentController:(PTDocumentController *)documentController bookmarksDidChange:(NSString *)bookmarkJson;
+- (void)documentController:(PTDocumentController *)documentController annotationsAsXFDFCommand:(NSString *)xfdfCommand;
+- (void)documentController:(PTDocumentController *)documentController documentLoadedFromFilePath:(NSString *)filePath;
+- (void)documentController:(PTDocumentController *)documentController documentError:(nullable NSError *)error;
+- (void)documentController:(PTDocumentController *)documentController annotationsChangedWithActionString:(NSString *)actionString;
+- (void)documentController:(PTDocumentController *)documentController annotationsSelected:(NSString *)annotations;
+- (void)documentController:(PTDocumentController *)documentController formFieldValueChanged:(NSString *)fieldString;
+- (void)documentController:(PTDocumentController *)docVC behaviorActivated:(NSString *)behaviorString;
+- (void)documentController:(PTDocumentController *)docVC leadingNavButtonClicked:(nullable NSString *)nav;
+- (void)documentController:(PTDocumentController *)docVC longPressMenuPressed:(NSString *)longPressMenuPressedString;
+- (void)documentController:(PTDocumentController *)docVC annotationMenuPressed:(NSString *)annotationMenuPressedString;
+- (void)documentController:(PTDocumentController *)docVC leadingNavButtonClicked:(nullable NSString *)nav;
+- (void)documentController:(PTDocumentController *)docVC pageChanged:(NSString *)pageNumbersString;
+- (void)documentController:(PTDocumentController *)docVC zoomChanged:(NSNumber *)zoom;
+- (void)documentController:(PTDocumentController *)docVC pageMoved:(NSString *)pageNumbersString;
+- (void)documentController:(PTDocumentController *)docVC scrollChanged:(NSString*)scrollString;
```

Comment:
I don't really understand these (and similar) changes. @terryyoon-pdftron any ideas? I guess this is also from before you took over.